frontend/src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,49 @@
+import { DialogInput, ActionsContainer } from './style';
```

Comment:
@yishain11 use `import * as SC from './style'` and add `SC.` prefix to components